### PR TITLE
[Fix bug 1259453] Remove extra space from links

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: mozillians/jinja2/groups/includes/group_edit_invitation.html [hunk 1]
```diff
@@ -19,11 +19,11 @@
                 <div class="col-sm-8">
 
                   <div class="avatar">
+                    <img class="profile-photo"
+                        src="{{ redeemer.get_photo_url('70x70') }}"
+                        alt="{{ _('Profile Photo') }}"
+                        title={{ redeemer.display_name }}>
                     <a href="{{ url('phonebook:profile_view', redeemer.user.username) }}">
-                      <img class="profile-photo"
-                          src="{{ redeemer.get_photo_url('70x70') }}"
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_invitation.html]: Please indent. (eg. ``src`` should be in the same column with ``class``).

**Comment by johngian** [comment]
r+wc

**Comment by johngian** [comment]
FYI you could have used this instead which is the jinja2 way to fix that kind of issues.
http://jinja.pocoo.org/docs/dev/templates/#whitespace-control

**Pull request merged**